### DISCUSSION TITLE
Everyone here vote for bringing back the old code search, comment with agree because they keep ignoring us even if we posted a lot of comments

### DISCUSSION THREAD

**Discussion by wokination** [community/community#55502] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Microsoft keep ruining their services one by one, if you wanna go well you need to hear from us, us who makes github grow, there's nothing powerful more than the power of the customers.

I NEED EVERYONE HERE TO VOTE FOR BRINGING BACK THE OLD CODE SEARCH AND REMOVING THIS BETA-JOKE

**Comment by yatinurhayati** [comment]
Agree

**Comment by pendy99** [comment]
Agree

**Reply by elblasy33** [reply]
Agree

**Comment by sixgodfan** [comment]
Agree

**Comment by Paladin1412** [comment]
agree

**Comment by lmnl01** [comment]
agree
